src/extension/app/constants.js

```diff
@@ -49,6 +49,7 @@ export const EXTERNAL_EVENTS = {
   RESOURCE_UNPUBLISHED: 'unpublished',
   EVIRONMENT_SWITCHED: 'envswitched',
   PLUGIN_USED: 'pluginused',
+  SIDEKICK_READY: 'sidekick-ready',
```

Comment:
@rofe should we change to `sidekickready` to be consistant with the other external events?

Reply:
I would probably keep it as is to reduce migration efforts of existing custom code. It's also the only one fired on the document vs the sidekick elelment.